File: examples/python/conditional_loop.py
# Simple loop for testing conditional breakpoints
# When using condition "i > 5", should only stop when i=6,7,8,9

def main():
    total = 0
    for i in range(10):
        value = i * 2  # Line 7: Set conditional breakpoint here
        total += value
        print(f"i={i}, value={value}, total={total}")

    print(f"Final total: {total}")
    return total

if __name__ == "__main__":
    result = main()

File: examples/javascript/conditional_loop.js
// Simple loop for testing conditional breakpoints
// When using condition "i > 5", should only stop when i=6,7,8,9

function main() {
    let total = 0;
    for (let i = 0; i < 10; i++) {
        const value = i * 2;  // Line 7: Set conditional breakpoint here
        total += value;
        console.log(`i=${i}, value=${value}, total=${total}`);
    }

    console.log(`Final total: ${total}`);
    return total;
}

main();
